Makefile.build

```diff
@@ -1,4 +1,5 @@
 export GOPROXY=https://proxy.golang.org
+export CGO_ENABLED=0
```

Comment:
What's this needed for?

Reply:
We use CGO_ENABLED=0 in goreleaser so this change will match the configuration that we release with.  This will bring down the build time a little bit.